packages/scaffold-ui/src/utils/w3m-email-otp-widget/styles.ts

```diff
@@ -4,4 +4,8 @@ export default css`
   wui-loading-spinner {
     margin: 9px auto;
   }
+
+  wui-text.email-text {
+    max-width: 300px;
```

Comment:
Why do we prefer max width here? Is it designer's decision? IMO we should avoid static width/heights and let the gaps (margin/padding) handle the responsiveness. AFAIK the modal width have 360px so maybe the parent element might have horizontal padding of `--wui-spacing-xl`?

Reply:
I believe this is because you need to set a specific width in order to have the ellipsis work

Reply:
Correct. You need to have a maximum inline width for ellipsis to kick in.

But you have a point, adding some padding might work, but I'm careful not to affect other aspects of the UI right now.

Reply:
The line-clamp shouldn't depend on the width of the elements. It should work dynamically. You could have a look in Tailwind's examples (by playing with card's width):
- https://tailwindcss.com/docs/line-clamp#basic-usage

But if it's still a case, I'm okay with the decision. Just let avoid static sizing.

Reply:
+ we are rendering the modal full width of the screen with `(max-width: 430px)` media query. This makes it even though we have enough space, the email will be truncated, which doesn't sound good IMO. Could we double check for mobile? @segunadebayo 🙏

Reply:
Refactored the code to use container based styling instead.

![CleanShot 2024-07-02 at 12 09 56@2x](https://github.com/WalletConnect/web3modal/assets/6916170/f87c94ad-c5db-4e68-a81f-f46b1bc834b4)